satpy/tests/reader_tests/test_ahi_hsd.py

```diff
@@ -359,6 +359,22 @@ def test_blocklen_error(self, *mocks):
                 fh._check_fpos(fp_, fpos, 0, 'header 1')
                 assert len(w) > 0
 
+    def test_time_rounding(self):
+        """Test rounding of the nominal time."""
+
+        assert AHIHSDFileHandler._is_valid_timeline(FAKE_BASIC_INFO['observation_timeline']) is True
+        assert AHIHSDFileHandler._is_valid_timeline('65526') is False
```

Comment:
These two assertions are independent of the rest and could be moved into a separate test method.

Reply:
Done!